packages/theme/example/index.html

```diff
@@ -310,23 +320,23 @@ <h1 id="buttons">Buttons</h1>
             <p id="buttons-left-icons" class="bs-component">
               <a href="#" class="btn btn-default">
                 <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><title>plus-circle</title><path d="M8,0a8,8,0,1,0,8,8A8,8,0,0,0,8,0ZM8,14a6.01658,6.01658,0,0,1-.85809-.0683c-.05658-.00818-.11279-.016-.16882-.02576a5.95226,5.95226,0,0,1-.821-.199l-.02319-.00848a5.9499,5.9499,0,0,1-.79907-.33118l-.04388-.02112a6.01748,6.01748,0,0,1-2.632-2.632l-.02112-.04388a5.9499,5.9499,0,0,1-.33118-.79907L2.29309,9.848a5.95226,5.95226,0,0,1-.199-.821c-.00977-.056-.01758-.11224-.02576-.16882A5.95773,5.95773,0,0,1,8.85809,2.0683c.05658.00818.11279.016.16882.02576a5.95226,5.95226,0,0,1,.821.199l.02319.00848a5.9499,5.9499,0,0,1,.79907.33118l.04388.02112a6.01748,6.01748,0,0,1,2.632,2.632l.02112.04388a5.9499,5.9499,0,0,1,.33118.79907l.00848.02319a5.95226,5.95226,0,0,1,.199.821c.00977.056.01758.11224.02576.16882A5.95773,5.95773,0,0,1,8,14Zm3-5H9v2H7V9H5V7H7V5H9V7h2Z"/></svg>
-                Default
+                <span>Default</span>
```

Comment:
The problem here is that it will not work without span and Bootstrap button are not designed to including `span` each time. (I think about some angular.js powered buttons or simple Bootstrap use)